.github/workflows/ci.yml

```
@@ -49,6 +49,25 @@ jobs:
         run: npm run test:unit
         continue-on-error: false
+        env:
+          NODE_ENV: test
+          CI: true
+          TURSO_DATABASE_URL: file:test.db
+          TURSO_AUTH_TOKEN: test-auth-token
+          METRICS_API_KEY: test-metrics-key
+          ADMIN_API_KEY: test-admin-key
+          SENTRY_DSN: https://test@sentry.io/test
+          VERCEL_URL: http://localhost:3000
+        env:
```

actionlint (1.7.7)

key "env" is duplicated in element of "steps" section. previously defined at line:52,col:9

(syntax-check)